test-data/unit/fine-grained-modules.test

```diff
@@ -1008,30 +1016,51 @@ import b
 b.foo(0)
 [file b.py.2]
 def foo() -> None: pass
+
+[stale b]
+[rechecked a, b]
+
 [file b.py.3]
 def foo(x: int) -> None: pass
+
+[stale2 b]
+
 [out]
 ==
 a.py:2: error: Too many arguments for "foo"
 ==
 
-[case testAddModuleAfterCache3]
+[case testAddModuleAfterCache3-skip-nocache]
 # cmd: mypy main a.py
 # cmd2: mypy main a.py b.py c.py d.py e.py f.py g.py
 # flags: --ignore-missing-imports --follow-imports=skip
 import a
 [file a.py]
 import b, c, d, e, f, g
+b.foo(10)
 [file b.py.2]
+def foo() -> None: pass
 [file c.py.2]
 [file d.py.2]
 [file e.py.2]
 [file f.py.2]
 [file g.py.2]
+
+-- No files should be stale or reprocessed in the first step since the large number
+-- of missing files will force build to give up on cache loading.
+[stale]
```

Comment:
Should we check for `[rechecked` here as well?

Reply:
When there is no `rechecked`, the value of `stale` is used. This is behavior inherited from the regular incremental test suite. I can document this better in the block comment or just add all the recheckeds in.